astronomer/providers/dbt/cloud/example_dags/example_dbt_cloud.py

```diff
@@ -0,0 +1,65 @@
+"""Example use of DBTCloudAsync related providers."""
+
+import os
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.providers.dbt.cloud.operators.dbt import DbtCloudRunJobOperator
+from airflow.utils.timezone import datetime
+
+from astronomer.providers.dbt.cloud.operators.dbt import DbtCloudRunJobOperatorAsync
+from astronomer.providers.dbt.cloud.sensors.dbt import DbtCloudJobRunSensorAsync
+
+DBT_CLOUD_CONN_ID = os.getenv("ASTRO_DBT_CLOUD_CONN", "dbt_cloud_default")
+DBT_CLOUD_ACCOUNT_ID = os.getenv("ASTRO_DBT_CLOUD_ACCOUNT_ID", 12345)
+DBT_CLOUD_JOB_ID = int(os.getenv("ASTRO_DBT_CLOUD_JOB_ID", 12345))
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
+
+
+default_args = {
+    "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
+    "dbt_cloud_conn_id": DBT_CLOUD_CONN_ID,
+    "account_id": DBT_CLOUD_ACCOUNT_ID,
+    "retries": int(os.getenv("DEFAULT_TASK_RETRIES", 2)),
+    "retry_delay": timedelta(seconds=int(os.getenv("DEFAULT_RETRY_DELAY_SECONDS", 60))),
+}
+
+with DAG(
+    dag_id="example_dbt_cloud",
+    start_date=datetime(2022, 1, 1),
+    schedule_interval=None,
+    default_args=default_args,
+    tags=["example", "async", "dbt-cloud"],
+    catchup=False,
+) as dag:
+    begin = EmptyOperator(task_id="begin")
```

Comment:
no strong opinion but we should name it `start`

Reply:
ok fixed